### DISCUSSION TITLE
Qiitaの記事編集画面でツールボックスを表示

### DISCUSSION THREAD

**Discussion by hellowork-mhlw** [increments/qiita-discussions#393] [要望・アイデア・不具合]
GitHubやRedmineみたいに編集画面で太字にするボタンやショートカットがあるといいと思います。
マークダウン初学者には、毎回MDの使い方を調べないといけなくなり大変です。summary/detailsやコードはよく使います。

![image](https://user-images.githubusercontent.com/116468267/209468963-6f2b2e68-e238-44c3-aabd-63cddf9f4051.png)
![image](https://user-images.githubusercontent.com/116468267/209468943-a1af6a60-a740-43c1-bd1a-8a61edde525e.png)

**Comment by tanaka-asami415** [comment]
@hellowork-mhlw 
ご意見ありがとうございます🙏
参考にさせていただきます。
